students/KellyM/Lesson2/grid.py

```diff
@@ -32,4 +32,9 @@ def variable_grid(x,y):
 		    print((y*(bar + (y*' ')) + bar))
 	    
     print((y*(plus + (y*minus)) + plus))	
+<<<<<<< HEAD
+  
+    
+=======
  
+>>>>>>> c22db40b2005d59325404152d7b47aa6085fabc3
```

Comment:
this is a diff from git when it's trying to merge files, you want to avoid committing those